test/browser.js

```diff
@@ -1,68 +0,0 @@
-import test from 'ava'
```

Comment:
I removed this test because it runs in jsdom and doesn't work with React 16. 
Someday we'll need to setup proper e2e tests. I don't have the time for it now though